### DISCUSSION TITLE
关于目前新版本备份盘和资源盘的问题

### DISCUSSION THREAD

**Discussion by jwister** [messense/aliyundrive-webdav#877] [Q&A] [answered]
默认挂载的好像是备份盘的东西，但是现在转存的资源都只能在资源盘。有解决有解决方案能够直接挂载资源盘的内容吗？

**Comment by messense** [comment, marked as answer]
2.3.0 已支持，增加 `--drive-type resource`.

**Reply by gly11** [reply]
补充一下，在docker中用`-e DRIVE_TYPE=resource`添加，docker-compose相应的添加在`environment`中即可

**Comment by jwister** [comment]
您的邮件已收到，谢谢。
